content/en/docs/components/katib/env-variables.md

```diff
@@ -176,20 +176,46 @@ deployment:
         <td>katib</td>
         <td>No</td>
       </tr>
+      <tr>
+        <td><code>KATIB_POSTGRESQL_DB_HOST</code></td>
+        <td>Katib Postgres Host</td>
+        <td>katib-postgres</td>
+        <td>No</td>
+      </tr>
+      <tr>
+        <td><code>KATIB_POSTGRESQL_DB_PORT</code></td>
+        <td>Katib Postgres Port</td>
+        <td>5432</td>
+        <td>No</td>
+      </tr>
+      <tr>
+        <td><code>KATIB_POSTGRESQL_DATABASE</code></td>
+        <td>Katib Postgres Database name</td>
+        <td>katib</td>
+        <td>No</td>
+      </tr>
     </tbody>
   </table>
 </div>
 
-Currently, Katib DB Manager supports only **MySQL** database. You can use your own DB Manager and Database to report metrics.
+Currently, Katib DB Manager supports only **MySQL** and **Postgres** database. (`DB_NAME` env variable must be filled with one of `mysql` or `postgres`)  
+However, you can use your own DB Manager and Database to report metrics by implements the [katib db interface](https://github.com/kubeflow/katib/blob/master/pkg/db/v1beta1/common/kdb.go).
 
 For the [Katib DB Manager](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/components/db-manager/db-manager.yaml#L25)
-you can change `DB_PASSWORD` to your own MySQL DB password.
-
-Katib DB Manager creates connection to the DB, using `mysql` driver and this data source name:
+you can change `DB_PASSWORD` to your own DB password.
 
+Katib DB Manager creates DB connection to the DB by the type of DB.  
+If `DB_NAME=mysql`, it uses `mysql` driver and this data source name:  
 **`DB_USER:DB_PASSWORD@tcp(KATIB_MYSQL_DB_HOST:KATIB_MYSQL_DB_PORT)/KATIB_MYSQL_DB_DATABASE?timeout=5s`**
 
-## Katib MySQL DB
+If `DB_NAME=postgres`, it uses `pq` driver and this data source name:  
+**`postgresql://[DB_USER[:DB_PASSWORD]@][KATIB_POSTGRESQL_DB_HOST][:KATIB_POSTGRESQL_DB_PORT][/KATIB_POSTGRESQL_DB_DATABASE]`**
+
+## Katib DB
+
+Katib DB components supports MySQL and Postgres.
+
+#### Katib MySQL
```

Comment:
Should we use `###` instead of `####`?

Reply:
Yes, that would meet the convention. Thanks :)